index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="index.css">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio-Project</title>
+</head>
+
+<body>
+    <header class="navbar">
+        <a class="logo" href="#">My Logo</a>
+        <nav>
+            <div class="navbarItems">
+            <a href="#">Portoflio</a>
+            <a href="#">About</a>
+            <a href="#">Contact</a>
+            </div>
```

Comment:
- [OPTIONAL] Good job adding more than required for this milestone, I suggest using an unordered list to layout this menu in order to meet the semantic HTML rules

Reply:
Hi @GedeonTS,

Thank you for your kind feedback. I shall make the optional changes recommended by you. I shall make the git merge now.

Regards